src/components/Heading/Heading.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../themes/nice/nice.scss';
+
+.Heading {
+    font: {
+        family: var(--heading-font-family);
+        weight: var(--font-weight-normal);
```

Comment:
I think this should be themeble, some people might like bold headings and possibly colored...